src/Runner.Common/Util/NodeUtil.cs

```diff
@@ -15,8 +15,14 @@ public static class NodeUtil
 
         public static string GetInternalNodeVersion()
         {
-            var forcedNodeVersion = Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedInternalNodeVersion);
-            return !string.IsNullOrEmpty(forcedNodeVersion) && BuiltInNodeVersions.Contains(forcedNodeVersion) ? forcedNodeVersion : _defaultNodeVersion;
+            var forcedInternalNodeVersion = Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedInternalNodeVersion);
```

Comment:
Is this just a refactor? It doesn't appear to do anything if i'm reading it correctly. Should we just revert these changes?

Reply:
@fhammerl asked to refactor this one with meaningful var names in [this](https://github.com/actions/runner/pull/1913#discussion_r886521138) comment